python/src/vmaf/config.py

```diff
@@ -20,7 +20,7 @@ def _path_from_external(name):
         :return str: Configured path, if any
         """
         try:
-            import externals
+            from . import externals
```

Comment:
Forget about stuff like this -> with python3 all imports must be absolute, and one has to explicitly use `.` to do a relative import.